mhs-reference-implementation/mhs/SDSLookup/sds.py

```diff
@@ -0,0 +1,105 @@
+import ssl
+from pathlib import Path
+
+from definitions import ROOT_DIR
+import ldap3
+
+from tornado.ioloop import IOLoop
+
+NHS_SERVICES_BASE = "ou=services, o=nhs"
+
+MHS_OBJECT_CLASS = "nhsMhs"
+AS_OBJECT_CLASS = "nhsAs"
+
+# These need replacing as the appropriate TLS certs are received for PTL
+PRIVATE_KEY = Path(ROOT_DIR) / 'data' / 'certs' / 'client.key'
+LOCAL_CERT_FILE = Path(ROOT_DIR) / 'data' / 'certs' / 'client.cert'
+CA_CERTS_FILE = Path(ROOT_DIR) / 'data' / 'certs' / 'client.cert'
+
+
+class SDSClient:
+    mhs_attributes = [
```

Comment:
Any reason these are a class variable, rather than a constant defined outside of the class?

Reply:
Nope, I've moved them outside the class